autometa/validation/download_dataset.py

```diff
@@ -0,0 +1,47 @@
+# in response to issue #110
+# Intended: fetch data similarly to scikit-learn API
```

Comment:
Please make this file executable with `chmod +x autometa/validation/download_dataset.py`, then do a commit.

Reply:
I had no idea git preserved permissions